far/findfile.cpp

```diff
@@ -3137,21 +3146,19 @@ FindFiles::FindFiles():
 			FindAskDlg[FAD_CHECKBOX_STREAMS].Selected = Global->Opt->FindOpt.FindAlternateStreams;
 		}
 		if (!(FindAskDlg[FAD_CHECKBOX_ARC].Flags & DIF_DISABLE))
-			FindAskDlg[FAD_CHECKBOX_ARC].Selected=SearchInArchives;
+			FindAskDlg[FAD_CHECKBOX_ARC].Selected = Options.SearchInArchives;
 
 		FindAskDlg[FAD_EDIT_MASK].strData = strFindMask;
 
-		if (SearchHex)
+		if (Options.SearchHex)
 			FindAskDlg[FAD_EDIT_HEX].strData = strFindStr;
 		else
 			FindAskDlg[FAD_EDIT_TEXT].strData = strFindStr;
 
-		FindAskDlg[FAD_CHECKBOX_CASE].Selected =
-			SearchCaseFold == search_case_fold::exact? BSTATE_CHECKED :
-			SearchCaseFold == search_case_fold::icase? BSTATE_UNCHECKED :
-			BSTATE_3STATE;
-		FindAskDlg[FAD_CHECKBOX_WHOLEWORDS].Selected=WholeWords;
-		FindAskDlg[FAD_CHECKBOX_HEX].Selected=SearchHex;
+		FindAskDlg[FAD_CHECKBOX_CASE].Selected = Options.CaseSensitive;
+		FindAskDlg[FAD_CHECKBOX_WHOLEWORDS].Selected = Options.WholeWords;
+		FindAskDlg[FAD_CHECKBOX_FUZZY].Selected = Options.Fuzzy;
+		FindAskDlg[FAD_CHECKBOX_HEX].Selected = Options.SearchHex;
 		const auto Dlg = Dialog::create(FindAskDlg, &FindFiles::MainDlgProc, this);
 		Dlg->SetAutomation(FAD_CHECKBOX_FILTER,FAD_BUTTON_FILTER,DIF_DISABLE,DIF_NONE,DIF_NONE,DIF_DISABLE);
 		Dlg->SetHelp(L"FindFile"sv);
```

Comment:
Please change 21 to 22 in `SetPosition` call below.

Reply:
Ugh! Thank you! Missed this.